src/Command/App/NewFromDrupal7Command.php

```diff
@@ -148,6 +148,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return Command::FAILURE;
       }
     }
+    // PHP defaults to no user agent. (Drupal.org's) GitLab requires it.
+    // @see https://www.php.net/manual/en/filesystem.configuration.php#ini.user-agent
+    ini_set('user_agent', 'ACLI');
```

Comment:
Guzzle HTTP client is the idiomatic way of making web requests in ACLI. For example: https://github.com/acquia/cli/blob/91306ec347d7f106ff3aeb5c924d1a029a0a2409/src/Command/Pull/PullCommandBase.php#L250

I opened CLI-1160 and CLI-1161 to set the user agent for Guzzle requests and replace instances of fopen with Guzzle. Let me know if you anticipate any problems with that.